src/filesystem.cc

```diff
@@ -70,12 +70,21 @@
 #include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+
 #include <cerrno>
 #include <fstream>
+#include <mutex>
+
 #include "constants.h"
 #include "status.h"
 #include "triton/common/logging.h"
 
+#define TRITONJSON_STATUSTYPE triton::core::Status
+#define TRITONJSON_STATUSRETURN(M) \
+  return triton::core::Status(triton::core::Status::Code::INTERNAL, (M))
+#define TRITONJSON_STATUSSUCCESS triton::core::Status::Success
```

Comment:
Are these being used?

Reply:
Yes, removing them will cause compiler error. This is just how other files import `triton_json.h`.